changes.go

```diff
@@ -780,11 +780,17 @@ func (s *ChangesService) change(tail string, changeID string, input interface{})
 
 	v := new(ChangeInfo)
 	resp, err := s.client.Do(req, v)
+	if err != nil {
+		return nil, resp, err
+	}
 	if resp.StatusCode == http.StatusConflict {
-		body, _ := ioutil.ReadAll(resp.Body)
-		err = errors.New(string(body[:]))
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return v, resp, err
+		}
+		return v, resp, errors.New(string(body[:]))
```

Comment:
A trivial thing I just spotted: `body[:]` could be simplified to `body`. It's already a slice, no need to slice it again.